gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -1,17 +1,18 @@
-# FIXME:
-#   1. Edit foreman-proxy requirement and specify minimum version
-#   2. Delete these lines
-#
+# Generated from <%= package.spec.file_name %> by gem2rpm -*- rpm-spec -*-
 # template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name <%= spec.name %>
 %global plugin_name <%= spec.name.sub(/\Asmart_proxy_/, '') %>
 
-%global foreman_proxy_min_version FIXME
-%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir /usr/share/foreman-proxy
```

Comment:
Why this change? I assume that this directory is different within the SCL but I'd prefer a line similar to the one we have in various other places:
```
%{!?_root_datarootdir:%global _root_datarootdir %{_datarootdir}}
```

Reply:
You are correct in your assumption. Can you point me to where this line is used? I get you want to be programmatic about things, but I find this more confusing them simply stating the foreman_proxy_dir is at /usr/share/foreman-proxy. Maybe this changes on something like EL8, but guessing not and is way easier to comprehend.

Reply:
We don't do it for datarootdir but there is for sysconfdir:
https://github.com/theforeman/foreman-packaging/blob/e3312eef30898896c3ff427dd22523bb92e4564e/gem2rpm/hammer_plugin.spec.erb#L13-L14

Reply:
Could you have a look at this? I can also easily be convinced to only define `foreman_proxy_bundlerd_dir` since only 2 spec files actually use `foreman_proxy_dir`.